src/Image.java

```diff
@@ -0,0 +1,17 @@
+public class Image extends ProductAttribute {
```

Comment:
Images maybe

Reply:
It is one photo, not an array. Why should it be Images?